src/core-asyncpipeline/AsyncPipeline/AsyncResultFlow/Map/Map.Task.cs

```diff
@@ -0,0 +1,48 @@
+#nullable enable
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System
+{
+    partial struct AsyncResultFlow<TSuccess, TFailure>
+    {
+        public AsyncResultFlow<TResultSuccess, TResultFailure> Map<TResultSuccess, TResultFailure>(
```

Comment:
I suppose _async_ `Map/MapValue` funcs should have the only overload with both _async_ `mapSuccessAsync/mapFailureAsync` strategies.

That's just mapping, not forwarding, etc.
The success and failure transforming look equal for mapping case.

So, let's reduce the overloads.
Whether a user needs for, e.g., for _async_ success mapping and _sync_ failure mapping (or vice versa) at the same time, they may use `[Value]Task.FromResult` for the the _sync_ strategy, or better _async_ `MapSuccess` and _sync_ `MapFailure` in series (or vice versa).

Reply:
Done